operator/controllers/kyma_controller.go

```diff
@@ -93,12 +99,24 @@ func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{}, r.Update(ctx, &kyma)
 	}
 
+	syncContext, err := remote.InitializeKymaSynchronizationContext(ctx, r.Client, &kyma)
+	if err != nil {
+		return ctrl.Result{RequeueAfter: 5 * time.Second}, err
+	}
+	remoteKyma, err := syncContext.CreateOrFetchRemoteKyma(ctx)
+	if err != nil {
+		return ctrl.Result{RequeueAfter: 5 * time.Second}, err
+	}
+	if synchronizationRequiresRequeue, err := syncContext.SynchronizeRemoteKyma(ctx, remoteKyma); err != nil || synchronizationRequiresRequeue {
+		return ctrl.Result{RequeueAfter: 5 * time.Second}, err
+	}
```

Comment:
is there a reason why requeue interval here is set to 5sec. and 10sec. at other places?

Reply:
tbh this is arbitrary, i will unify it

Reply:
Introduced generic Intervals and splitted it into 3 variables for Success Requeue, Failure Requeue and Pending Requeue